helm/values-dev.yaml

```diff
@@ -87,6 +87,18 @@ microservice-chart:
     ROOT_LOGGING_LEVEL: "INFO"
     APP_LOGGING_LEVEL: "INFO"
     WEB_LOGGING_LEVEL: "OFF"
+    MONGO_HOST: pagopa-d-weu-ecommerce-cosmos-account.mongo.cosmos.azure.com
```

Comment:
missing MONGO_PASSWORD env property (to be add as a secret, see [here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/d50c375fdfe411bfc6988f4dea95f4811e02b353/helm/values-dev.yaml#L186) for an example).

Please add it to helm values for all envs